SimpleAnimationLab/app/src/main/java/com/justinwells/simpleanimationlab/MainActivity.java

```diff
@@ -0,0 +1,56 @@
+package com.justinwells.simpleanimationlab;
+
+import android.animation.LayoutTransition;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.animation.Animation;
+import android.view.animation.AnimationUtils;
+import android.widget.EditText;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+public class MainActivity extends AppCompatActivity {
+    private ViewGroup wishList;
+    private ImageView addWish, kazaam;
+    private EditText wish;
+    private TextView wishText;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        LayoutTransition layoutTransition = new LayoutTransition();
+        layoutTransition.enableTransitionType(LayoutTransition.CHANGING);
+        wishList = (ViewGroup) findViewById(R.id.wish_list);
+        wishList.setLayoutTransition(layoutTransition);
```

Comment:
Since you put android:animateLayoutChanges="true" in the XML for the LinearLayout, you actually don't need to also set up the layout transition animation here. That one line in XML accomplishes the same thing.